decisions/wc_religion_decisions.txt

```diff
@@ -115,8 +115,6 @@ decisions = {
 
 	# loa_group
 	pick_main_loa = {
-		is_high_prio = yes
-
```

Comment:
I wouldn't make it a non-high priority because it's basically a one-time decision and you don't pay anything.